src/components/general/Navbar/index.js

```diff
@@ -163,6 +163,7 @@ function Navbar(props) {
         if (confirm) {
             stopSiloTasks();
             logout();
+            localStorage.clear();
```

Comment:
Can we just clear the localStorage using keys that we used?

Reply:
Looks like it is not possible right now.
Still, there are certain keys that we store in localStorage that we should not clear.

Reply:
@AdityaKhatri suggested we just use static key for widgets and sections.

We can't save analysis framework anyway, so it doesn't matter if the data is shared between different analysis frameworks.

Reply:
I have added static keys and cleared only those keys during logout